### ISSUE TITLE
is any way add space to last of candle chart in javascript v3 amcharts 

### ISSUE THREAD

**Issue by vishnug1995** [amcharts/amcharts3#220]
![image](https://user-images.githubusercontent.com/31384107/49779306-5e2b8200-fd2f-11e8-82f5-1d117ebc5d67.png)

we want to add space in after last candle in candle charts v3

**Comment by martynasma** [comment]
The only way to do that is to add empty data points at the end.

Either that, or use Date Padding plugin:
https://github.com/amcharts/tools/tree/master/datePadding

**Comment by vishnug1995** [comment]
is that any npm packages available for datepadding ?

**Comment by martynasma** [comment]
No, I'm afraid. Plugins are not available as npm packages. Sorry.
